src/cli/config.rs

```diff
@@ -37,3 +39,10 @@ pub struct StateIdArg {
     )]
     pub state_id: StateId,
 }
+
+#[derive(Args, Debug, Clone)]
+pub struct ValidatorsArg {
+    pub state_id: StateId,
+    pub validator_id: PublicKeyOrIndex,
+    pub status: ValidatorStatus,
+}
```

Comment:
if you are adding this:

https://ethereum.github.io/beacon-APIs/?urls.primaryName=v2.4.1#/Beacon/getStateValidator

then I only see these params:

```suggestion
#[derive(Args, Debug, Clone)]
pub struct ValidatorsArg {
    pub state_id: StateId,
    pub validator_id: PublicKeyOrIndex,
}
```